apps/web/src/components/display/AddRecordModal.tsx

```diff
@@ -87,7 +87,7 @@ export const AddRecordModal: React.FC<AddRecordProps> = (props: AddRecordProps)
   };
 
   return (
-    <Modal open={visible} handleClose={close}>
+    <Modal open={visible} handleClose={() => void 0}>
```

Comment:
I'm curious about whats going on here. How does this function work?

Reply:
it's same as `undefined`. Modal requires handleClose prop but it isn't called ever and I don't know why.